code/modules/reqs/supplypacks.dm

```diff
@@ -679,53 +580,68 @@ ARMOR
 /datum/supply_packs/armor/masks
 	name = "SWAT protective mask"
 	contains = list(/obj/item/clothing/mask/gas/swat)
-	cost = 10
+	cost = 5
+
+/datum/supply_packs/armor/nvg
+	name = "Night Vision Goggles"
+	contains = list(/obj/item/clothing/glasses/night/m42_night_goggles)
+	cost = 50
+
+/datum/supply_packs/armor/thermal
+	name = "Thermal Goggles"
+	contains = list(/obj/item/clothing/glasses/thermal/m64_thermal_goggles)
+	cost = 60
```

Comment:
No.

Reply:
These were in before in Electric's PR. They make a difference, and their price reflects that. The former has a slightly bigger impact than a Baldur lamp, mesons and the like, while freeing up your module slot. The latter helps in tracking down those xenos that just keep running, and helps in stopping ambushes.

Reply:
@TiviPlus